### PR TITLE
Update realpath when file has been moved

### PR DESCRIPTION
**Pull request by MichelBahl**
if cameraRoll is set to true, an error occur on android due to moved image.

Fixed this issue by updating image path with path of the moved file.

### CODE REVIEW

File: Example/package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "adb-reverse": "adb reverse tcp:8081 tcp:8081"
   },
   "dependencies": {
-    "react": "15.4.1",
-    "react-native": "^0.40.0",
+    "react": "15.3.1",
+    "react-native": "^0.33.1",
```
Comment (marcshilling) [inline Example/package.json]: This should not be changed. We made a decision a while back to require RN 0.40 or greater.

**Comment by marcshilling** [comment]
Looks like we've got some conflicts based on recent changes.

@rusfearuth what are your thoughts on this?

**Comment by alexustinovsm** [comment]
@marcshilling Going to check it out and write answer today.

**Comment by rusfearuth** [comment]
@marcshilling Going to check it out and write answer today.

**Comment by rusfearuth** [comment]
@marcshilling Will it work on iOS? I see @MichelBahl downgrade react-native version.

**Pull request closed without merging**